src/components/Users/UserAvailabilityTab.tsx

```diff
@@ -85,6 +85,15 @@ export default function UserAvailabilityTab({ userData: user }: Props) {
     }),
   });
 
+  const { data: isSchedulableResource } = useQuery({
+    queryKey: ["practitioners", facilityId],
```

Comment:
the key is now no longer correct; `["practicioners", facilityId]` is meant for practicioners search; here we are selecting only if the user is schedulable resource;

let's have a different key;